tests/rptest/services/openmessaging_benchmark_configs.py

```diff
@@ -110,84 +110,108 @@ def validate_metrics(metrics, validator):
         "name": "simple-driver",
         "replication_factor": 3,
         "request_timeout": 300000,
-        "enable_idempotence": "false",
-        "acks": "1",
-        "linger_ms": 1,
-        "max_in_flight": 1,
-        "batch_size": 131072,
-        "auto_offset_earliest": "earliest",
-        "auto_commit": "false",
-        "max_partition_fetch": 131072
+        "producer_config": {
+            "enable.idempotence": "false",
+            "acks": "1",
+            "linger.ms": 1,
+            "max.in.flight.requests.per.connection": 1,
+            "batch.size": 131072,
+        },
+        "consumer_config": {
+            "auto.offset.reset": "earliest",
+            "enable.auto.commit": "false",
+            "max.partition.fetch.bytes": 131072
+        },
     }
 
     ACK_ALL_GROUP_LINGER_1MS = {
         "name": "ack-all-group-linger-1-ms-no-idem",
         "replication_factor": 3,
         "request_timeout": 300000,
-        "enable_idempotence": "false",
-        "acks": "all",
-        "linger_ms": 1,
-        "max_in_flight": 1,
-        "batch_size": 131072,
-        "auto_offset_earliest": "earliest",
-        "auto_commit": "false",
-        "max_partition_fetch": 131072
+        "producer_config": {
+            "enable.idempotence": "false",
+            "acks": "all",
+            "linger.ms": 1,
+            "max.in.flight.requests.per.connection": 1,
+            "batch.size": 131072,
+        },
+        "consumer_config": {
+            "auto.offset.reset": "earliest",
+            "enable.auto.commit": "false",
+            "max.partition.fetch.bytes": 131072
+        },
     }
 
     ACK_ALL_GROUP_LINGER_1MS_WITH_IDEMPOTENCE = {
         "name": "ack-all-group-linger-1-ms-enable-idem",
         "replication_factor": 3,
         "request_timeout": 300000,
-        "enable_idempotence": "true",
-        "acks": "all",
-        "linger_ms": 1,
-        "max_in_flight": 1,
-        "batch_size": 131072,
-        "auto_offset_earliest": "earliest",
-        "auto_commit": "false",
-        "max_partition_fetch": 131072
+        "producer_config": {
+            "enable.idempotence": "true",
+            "acks": "all",
+            "linger.ms": 1,
+            "max.in.flight.requests.per.connection": 1,
```

Comment:
Why would we set this to 1? Are these based on driver configs in OMB repo?

Reply:
It's based off of our commonly used defaults in ducktape. I could use the librdkafkas default of 5ms. Or the Java and franzo-go client default of 0ms if that is better.

Reply:
Sorry I was referring to `max.in.flight.requests.per.connection` not linger. Seems like we should probably not override this one?

Reply:
@ballard26 if there's anything we want change maybe let's open a follow up for this one so we can get this in.